### ISSUE TITLE
.MOV file portrait video displaying as landscape iOS 14

### ISSUE THREAD

**Issue by dylan-westbury** [TheWidlarzGroup/react-native-video#2158]
# Bug
.MOV video recorded on iPhone 11 / iPhone 11 pro max device in portrait is displaying as a landscape video on iOS 14. 

**This screenshot of iPhone 11 simulator with resizeMode as cover - shows as landscape then zooms in to fit screen** 
![Screen Shot 2020-09-23 at 9 53 12 am](https://user-images.githubusercontent.com/16129043/93949452-033c1100-fd84-11ea-8b6e-242928de032c.png)

**This screenshot of iPhone 11 simulator with resizeMode as none - you can see how it's displaying as landscape and not portrait** 
![Screen Shot 2020-09-23 at 10 06 02 am](https://user-images.githubusercontent.com/16129043/93949674-9aa16400-fd84-11ea-8542-6cd328ee7a83.png)

**This is screenshot of same video from iPhone 11 pro max iOS 13.7 - issue does not exist**
![IMG_3706](https://user-images.githubusercontent.com/16129043/93949124-7e50f780-fd83-11ea-99a5-fa1822f42b1d.PNG)

If I convert to MP4 the video shows fine

I also see someone commented here with same issue for mov 4.4.5 https://github.com/react-native-community/react-native-video/pull/842#issuecomment-660703541

## Platform
* iOS

## Environment info

React native info output:

```bash
    OS: macOS 10.15.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 396.60 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.16.3 - /usr/local/bin/node
    Yarn: 1.19.1 - ~/.yarn/bin/yarn
    npm: 6.14.7 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.0, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
    Android SDK:
      API Levels: 22, 23, 25, 26, 27, 28, 29
      Build Tools: 26.0.3, 27.0.3, 28.0.3, 29.0.2
      System Images: android-26 | Google Play Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
      Android NDK: 20.0.5594570
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5900203
    Xcode: 12.0/12A7209 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_181 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.2 => 0.63.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

```

Library version: 4.4.5 and 5.1.0-alpha8

## Steps To Reproduce

1. Record video on iphone 11 / iphone 11 pro max
2. Open with into app from camera roll (using library such as react-native-image-crop-picker)
3. Display video using react-native-video (video will be showing as landscape instead of portrait)
...

## Expected behaviour

1. Video should display as portrait if recorded as portrait
2.

## Reproducible sample code

```
<Video
            ref={ videoUseRef }
            paused={ paused }
            onEnd={ () => {
              onEnd();
              setEnded(true);
            } }
            onLoad={ onLoad }
            onProgress={ onProgress }
            source={ from === 'local' ? profileVideoUrl : { uri: profileVideoUrl } }
            style={{
    position: 'absolute',
    top: 0,
    left: 0,
    bottom: 0,
    right: 0,
    backgroundColor: '#000'
  } }
            resizeMode={ 'cover' } //or 'none'
            repeat={ false }
            ignoreSilentSwitch="ignore"
          />
```
## Video sample
https://drive.google.com/file/d/1c8qiVfRyhn4aKvY3wXd2u6Yt05qeCvX6/view?usp=sharing

**Comment by coschmit** [comment]
the same for me :(

**Comment by leshaivanovsky** [comment]
we face the same problem in our app, looking into that

**Comment by hey-stefan** [comment]
Same here. :(

**Comment by dylan-westbury** [comment]
@leshaivanovsky @stefan-t 

I followed the fix provided in https://github.com/react-native-community/react-native-video/issues/2159

Check where the ++++++ is (add those lines of code)

**Comment by hey-stefan** [comment]
Cheers @dylan-westbury! 
However, this fix introduces two new problems (as mentioned by [@2xSamurai](https://github.com/react-native-community/react-native-video/issues/2159#issuecomment-697292774) and [@dackom](https://github.com/react-native-community/react-native-video/issues/2159#issuecomment-698008305) in that issue).

**Comment by dylan-westbury** [comment]
@stefan-t 

I see, for our use case we don't use controls.

**Comment by albmin** [comment]
Out of curiosity, how did you convert the .mov to .mp4? programatically or manually as part of troubleshooting?

**Comment by dylan-westbury** [comment]
@albmin manually outside of application

**Comment by hamzaouhssain1993** [comment]
iOS 14.2 beta fixed the issue

**Comment by sachithkadamba** [comment]
> @albmin manually outside of application

hi, how did you convert to mp4? I tried with moviepy, but its very slow process.

**Comment by dylan-westbury** [comment]
Just first result I found on google @sachithkadamba

**Comment by harveyconnor** [comment]
yep same issue here

**Comment by antoarva** [comment]
Yesterday iOS 14.1 released, the video is fixed now. Just update.

**Comment by pvroosendaal** [comment]
Fixed in iOS 14.1 indeed.

**Comment by ddave09** [comment]
One work around I found was to use 'resizeMode: cover' but caveat with that is that you would have to compromise with the aspect ratio which can be a pretty big deal.

**Comment by leurs247** [comment]
My (temporary) solution is to convert a .MOV-file to an .MP4-file with FFMPEG. I use react-native-fs (copyAssetsVideoIOS) to make a copy. Then I use the following FFMPEG command to conver the .MOV-file to the .MP4-file:
````
-i input.MOV  -preset ultrafast -vcodec h264 -acodec mp2 output.MP4
````
After that, I delete the copied input.MOV file (``RNFS.unlink('input.MOV')`. I use the path to utput.MP4 to show the video. This video is showed correctly. Extra is that I now only have MP4-files on my server, from IOS and from Android.

**Comment by dylan-westbury** [comment]
Closing as resolved in iOS 14.1+

**Comment by nmhoan76** [comment]
> My (temporary) solution is to convert a .MOV-file to an .MP4-file with FFMPEG. I use react-native-fs (copyAssetsVideoIOS) to make a copy. Then I use the following FFMPEG command to conver the .MOV-file to the .MP4-file:
> 
> ```
> -i input.MOV  -preset ultrafast -vcodec h264 -acodec mp2 output.MP4
> ```
> 
> After that, I delete the copied input.MOV file (``RNFS.unlink('input.MOV')`. I use the path to utput.MP4 to show the video. This video is showed correctly. Extra is that I now only have MP4-files on my server, from IOS and from Android.

command don't active
